app/src/main/assets/OpenAPSSMB/determine-basal.js

```diff
@@ -920,10 +920,10 @@ var determine_basal = function determine_basal(glucose_status, currenttemp, iob_
                 rT.reason += ", but Min. Delta " + minDelta.toFixed(2) + " > Exp. Delta " + convert_bg(expectedDelta, profile);
             }
             if (currenttemp.duration > 15 && (round_basal(basal, profile) === round_basal(currenttemp.rate, profile))) {
-                rT.reason += ", temp " + currenttemp.rate + " ~ req " + round(basal, 2) + "U/hr. ";
+                rT.reason += ", temp " + currenttemp.rate + " ~ req " + basal + "U/hr. ";
```

Comment:
This (and a couple references below) seem like they are reverting ca0478572aaccb660f47d001a9f53b6fc614e341 - is that intended?